dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/impl/DatabaseHelper.java

```diff
@@ -99,6 +103,14 @@
      * SQLException SQL States that indicate a stale connection.
      */
     final Set<String> staleSQLStates = new HashSet<String>();
+    
+    /**
+     * Pairs of SQLException error codes and SQL states that indicate a stale exception.
+     * This differs from staleErrorCodes and staleSQLStates because here both items must
+     * match in order to be considered stale.
+     * The entries are of the format: SQLSTATE-SQLCODE
```

Comment:
The - is a confusing delimiter given that error codes can and sometimes do have negative values.  A  different delimiter (maybe &) would be clearer for someone who is debugging.

Reply:
good point, I'll change that to a forward-slash delimiter